plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -107,6 +110,13 @@ data CekValue uni fun =
       (CekValEnv uni fun)   -- Initial environment, used for evaluating every argument
     deriving (Show, Eq) -- Eq is just for tests.
 
+instance ToExMemory (CekValue uni fun) where
+    toExMemory (VCon mem _)               = mem
```

Comment:
Is there any particular reason for changing this from the `\case` version, or is it just a cosmetic thing?

Reply:
I think it got lost somewhere along the way. Any preference?

Reply:
No, it's up to you. I quite like using `\case` in situations like this, but I think that's something that's persisted from my OCaml experience.  I just wondered if there was some specific objection to `\case`.

Reply:
I like `\case` since I hate writing out the function name lots of times, but not a big deal.